src/blocks/product-card.scss

```diff
@@ -0,0 +1,96 @@
+.product-card {
+  width: $card-width;
+  padding: 32px 16px 16px;
+
+  border: 1px solid $border-color;
+  border-radius: 5px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+
+    margin: 0 auto;
+    margin-bottom: 40px;
+  }
+
+  &__main-title {
+    margin-bottom: 4px;
+
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+
+    color: $main-accent-color;
+  }
+
+  &__product-code {
+    margin-bottom: 16px;
+
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+
+    color: $secondary-accent-color;
+  }
+
+  &__rating,
+  &__price {
+    @include flex;
+    justify-content: space-between;
+  }
+
+  &__rating {
+    margin-bottom: 24px;
+  }
+
+  &__reviews {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+
+    color: $main-accent-color;
+  }
+
+  &__price {
+    margin-bottom: 16px;
+  }
+
+  &__price-title {
```

Comment:
The same thing, you can fold it and the price-value in &__price and gust &-title &-value